### PR TITLE
Deploy mobile to beta 

### PR DESCRIPTION
**Pull request by frabago**
QA list:
- Test the app normally to see if nothing is broken

Maybe you need to run this command on the Android directory if you have an error about fastlane:
- sudo gem install fastlane -NV

### CODE REVIEW

**Comment by dsusviela** [comment]
Everything seems to be working well.

File: src/components/Bale/TabletBale.js [hunk 1]
```diff
@@ -1,50 +1,74 @@
...
-import { View, Image } from 'react-native';
-import { Text, ListItem } from 'react-native-elements';
+import { View, Image, StyleSheet, Text } from 'react-native';
 import FardoIcon from '../../assets/images/FardoIcon.png';
 import Button from '../common/Button';
 import strings from '../../localization';
-import styles from './styles';
+import Colors from '../../helpers/Colors';
+
+const stylesTabletItem = StyleSheet.create({
```
Comment (dsusviela) [inline src/components/Bale/TabletBale.js]: Move the styles to the corresponding styles.js please

File: src/components/common/BaleList.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import React, { Component } from 'react';
```
Comment (dsusviela) [inline src/components/common/BaleList.js]: This file should be under Bale and not under common. I know this is not on your part but i just realize about it

File: android/app/src/main/java/com/reactnativetemplate/MainApplication.java [hunk 3]
```diff
@@ -65,9 +75,9 @@ public boolean clearHostOnActivityDestroy() {
...
     }
 
-    @Override
+   /*  @Override
     public void onCreate() {
         super.onCreate();
-        SoLoader.init(this, /* native exopackage */ false);
-    }
+        SoLoader.init(this, native exopackage false);
+    } */
```
[the remark this answers is not in the document]
Comment (frabago) [reply android/app/src/main/java/com/reactnativetemplate/MainApplication.java]: Deleted

File: src/components/Bale/TabletBale.js (hunk 1 above)
Comment (frabago) [reply src/components/Bale/TabletBale.js]: Moved to style

**Pull request merged**